cluster-logging.md

```diff
@@ -57,53 +57,47 @@ The "Storage" column shows whether the attribute is stored into a LokiStack usin
 * `stream label` (with an optional "required", if the Loki Operator will enforce this attribute in the configuration)
 * `structured metadata`
 
-| Name | Location | Applicable Sources | Storage (LokiStack) | Comment |
-| :--- | :------- | :----------------- | :------------------ | :------ |
-| `log_source` | resource | all | required stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `openshift.log.source` |
-| `log_type` | resource | all | required stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `openshift.log.type` |
-| `kubernetes.container_name` | resource | container | stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `k8s.container.name` |
-| `kubernetes.host` | resource | all | stream label | **(DEPRECATED)** Compatibility attribute, same information as `k8s.node.name` |
-| `kubernetes.namespace_name` | resource | container | required stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `k8s.namespace.name` |
-| `kubernetes.pod_name` | resource | container | stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `k8s.pod.name` |
-| `openshift.cluster_id` | resource | all | | **(DEPRECATED)** Compatibility attribute, contains same information as `openshift.cluster.uid` |
-| `level` | log | container, journal | | **(DEPRECATED)** Compatibility attribute, contains same information as `severityText` |
-| `openshift.cluster.uid` | resource | all | required stream label | |
-| `openshift.log.source` | resource | all | required stream label | |
-| `openshift.log.type` | resource | all | required stream label | |
-| `openshift.labels.*` | resource | all | structured metadata | |
-| `k8s.node.name` | resource | all | stream label | |
-| `k8s.namespace.name` | resource | container | required stream label | |
-| `k8s.container.name` | resource | container | stream label | |
-| `k8s.pod.labels.*` | resource | container | structured metadata | |
-| `k8s.pod.name` | resource | container | stream label | |
-| `k8s.pod.uid` | resource | container | structured metadata | |
-| `k8s.cronjob.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
```

Comment:
These were intentionally added because they are provided by the OTEL Collector, possibly in future by vector.  This may need some other indicator that they are not always present